doc/how-to-create-an-iceoryx2-release.md

```diff
@@ -79,10 +79,11 @@ number is `Xold.Yold.Zold`.
 10. Adjust the `<version>` to `X.Y.Z` in `$GIT_ROOT$/package.xml`.
 11. Call `rg "Xold\.Yold\.Zold"` and adjust all findings.
     * C and C++ examples, `BUILD.bazel` & `CMakeLists.txt`
-12. **Merge all changes to `main`.**
-13. Set tag on GitHub and add the release document as notes to the tag
+12. Adjust the major, minor and patch version number in `iceoryx2_bb_elementary::PackageVersion`
```

Comment:
Having a script taking care of setting the version number in all known locations would be helpful. I'll add it to my todo list :)